### ISSUE TITLE
[Issue]: Trickplays ignore display aspect ratio

### ISSUE THREAD

**Issue by mrnoname1000** [jellyfin/jellyfin#11785]
### Please describe your bug

When viewing a generated trick play for a video with a DAR of anything but 1:1, the displayed trickplay ignores it. It seems ffmpeg does correctly embed the DAR in the output jpg, and viewing it directly with MPV looks right, but the display in the browser is stretched nonetheless.

The solution could be implemented in either the client side (correctly displaying trickplay aspect ratio) or on the server side (pre-scaling trickplays, which would require regenerating for all videos without 1:1 DAR)

### Reproduction Steps

1. Find a video with a non-square DAR
1. Generate a trickplay for it
1. Hover over the progress bar and witness the stretched thumbnail

### Jellyfin Version

10.9.0

### Environment

```markdown
- OS: Debian 12.5
- Linux Kernel: 6.5
- Virtualization: Docker
- Clients: Browser, Desktop (JMP)
- Browser: Firefox
- FFmpeg Version: 6.0.1-jellyfin
- Playback Method: Direct play
- Hardware Acceleration: none
- GPU Model: none
- Plugins: Ani-Sync, AniDB, AniList, Artwork, AudioDB, Discogs, Fanart, InfuseSync, Intro Skipper, Kitsu, Kodi Sync Queue, Media Analyzer, Merge Versions, MusicBrainz, OMDb, One Pace, Open Subtitles, Playback Reporting, Reports, Session Cleaner, Studio Images, Subtitle Extract, TMDb, TheTVDB, Transcode Killer, VGMdb, Webhook
- Reverse Proxy: Caddy
- Base URL: none
- Networking: Docker bridge
- Storage: local
```

### Jellyfin logs

```shell
[10:45:33] [INF] [56] Jellyfin.Server.Implementations.Trickplay.TrickplayManager: Creating trickplay files at 320 width, for /media/shows/The IT Crowd/Season 2/It.Crowd.s02e03.mkv [ID: 41568ed4-03b2-9f36-fc9c-640e5c2088c4]
[10:45:33] [INF] [56] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Trickplay generation: /usr/lib/jellyfin-ffmpeg/ffmpeg -loglevel error -threads 0 -noautorotate -i file:"/media/shows/The IT Crowd/Season 2/It.Crowd.s02e03.mkv" -an -sn -vf "fps=0.20000000298023224,setparams=color_primaries=bt709:color_trc=bt709:colorspace=bt709,scale=trunc(min(max(iw\,ih*a)\,320)/2)*2:trunc(ow/a/2)*2,format=yuv420p" -threads 0 -c:v mjpeg -qscale:v 4 -f image2 "/cache/temp/a5aea42026674e97802b46255957997d/%08d.jpg"
[10:45:45] [INF] [101] Jellyfin.Server.Implementations.Trickplay.TrickplayManager: Finished creation of trickplay files for /media/shows/The IT Crowd/Season 2/It.Crowd.s02e03.mkv
```

### Please attach any screenshots here

<img width="1728" alt="Screenshot_2024_05_21_4_27_24_PM" src="https://github.com/jellyfin/jellyfin/assets/8191535/8530cc69-b076-45bb-b9d1-591cb057ce12">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by gnattu** [comment]
What's the SAR of this Video?

**Comment by mrnoname1000** [comment]
For the source, the SAR is 160:177 and the DAR 4:3. Looking more closely, the intermediate JPGs have SAR 9:10/DAR 4:3 and the final trickplays have SAR 1:1/DAR 40:27.

**Comment by Bond-009** [comment]
Should be fixed by #11798
